<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(194, 237, 206);</string>
  </property>
  <widget class="QPushButton" name="add_to_cart_btn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>410</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(102, 187, 51);</string>
   </property>
   <property name="text">
    <string>Add to Cart</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buy_now_btn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>410</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(111, 179, 184);</string>
   </property>
   <property name="text">
    <string>Buy Now</string>
   </property>
  </widget>
  <widget class="QLabel" name="prod_image_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>811</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 246, 242);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>761</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="descrip_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>771</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="delivery_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>761</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cash on Delivery</string>
   </property>
  </widget>
  <widget class="QLabel" name="price_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>description:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
